test/ex_machina/ecto_test.exs

```diff
@@ -127,6 +127,13 @@ defmodule ExMachina.EctoTest do
     assert TestRepo.one(User)
   end
 
+  test "save_record/1 ignores associations that are not set" do
+    article = Factory.save_record(%Article{title: "Ecto is Awesome"})
```

Comment:
What do you think about explicitly passing `Ecto.Association.NotLoaded` as the author? That way it's super clear what it's testing?